app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,261 @@
+<!--
+  ~ Copyright (c) 2019. Fulton Browne
+  ~  This program is free software: you can redistribute it and/or modify
+  ~     it under the terms of the GNU General Public License as published by
+  ~     the Free Software Foundation, either version 3 of the License, or
+  ~     (at your option) any later version.
+  ~
+  ~     This program is distributed in the hope that it will be useful,
+  ~     but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~     GNU General Public License for more details.
+  ~
+  ~     You should have received a copy of the GNU General Public License
+  ~     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+  -->
+
+<resources>
+    <string name="app_name">Ara</string>
+    <string name="action_settings">die Einstellungen</string>
+    <string name="action_about">Über</string>
+    <string name="greet">Hallo, kann ich helfen?</string>
```

Comment:
Usually you'd ask how you can help, so "Hallo, wie kann ich helfen?"

Reply:
Sure. Thanks for the review